manta/client.py

```diff
@@ -195,9 +199,13 @@ def _request(self, path, method="GET", query=None, body=None, headers=None):
                 headers["Date"] = http_date()
             sigstr = 'date: ' + headers["Date"]
             algorithm, fingerprint, signature = self.signer.sign(sigstr)
-            headers["Authorization"] = \
-                'Signature keyId="/%s/keys/%s",algorithm="%s",signature="%s"' % (
-                    self.account, fingerprint, algorithm, signature)
+            auth = ('Signature keyId="/{}/keys/{}",algorithm="{}",signature="{}"'
+                    .format('/'.join(filter(None, [self.account, self.subuser])),
+                            fingerprint, algorithm, signature))
+            headers["Authorization"] = auth
```

Comment:
Python 2.6 has been officially retired for more than 2 years now (and the last binary release for 2.6 was 7+ years ago). It receives no updates, security or otherwise. Are we sure we even want to still support 2.6 just to support RHEL6 users who haven't bothered to [update Python](http://www.curiousefficiency.org/posts/2015/04/stop-supporting-python26.html)?